tapa/steps/floorplan.py

```diff
@@ -22,6 +26,49 @@
     store_tapa_program,
 )
 
+AUTOBRIDGE_WORK_DIR = "autobridge"
+
+_logger = logging.getLogger().getChild(__name__)
+
+
+@click.command()
+@click.option(
+    "--device-config",
+    type=Path,
+    required=True,
+    help="Path to the device configuration file.",
+)
+@click.option(
+    "--floorplan-config",
+    type=Path,
+    required=True,
+    help="Path to the floorplan configuration file.",
+)
+def run_autobridge(device_config: Path, floorplan_config: Path) -> None:
+    """Run the autobridge tool to generate a floorplan."""
+    program = load_tapa_program()
+    ab_graph_path = Path(program.work_dir) / "ab_graph.json"
+    autobridge_work_dir = Path(program.work_dir) / AUTOBRIDGE_WORK_DIR
+    os.makedirs(str(autobridge_work_dir), exist_ok=True)
```

Comment:
```suggestion
    autobridge_work_dir.mkdir(parents=True, exist_ok=True)
```

Since we already are using `pathlib.Path`.